### PR TITLE
Fix 7 areas links v2

### PR DESCRIPTION
**Pull request by tunde-anddigital**
### Context
Issue 176 in JIRA: https://dfedigital.atlassian.net/jira/software/projects/HFEYP/boards/83?assignee=60016f8c9edf280075ecdd15&selectedIssue=HFEYP-176

### Changes proposed in this pull request
Implement JS from prototype into the app so that the boxes for the 7 areas of learning can also be clicked and direct the user to the relevant page.

### Guidance to review
- Navigate to the 7 areas of learning content page in the app. 
- Click on the boxes to ensure that navigation to the relevant pages works as required.

### CODE REVIEW

File: app/webpacker/packs/application.js [hunk 1]
```diff
@@ -15,10 +15,20 @@ $(document).ready(function() {
...
   });
-
+  
   //clipboard
   $('#clipboard_copier').click(function() {
     copyToClipboard();
   });
+
+  document.querySelectorAll('.nhsuk-card--clickable').forEach((panel) => {
+    // Check if panel has a link within it
```
Comment (guy-roberts) [inline app/webpacker/packs/application.js:25]: We will rename css classes beginning in nhsuk- shortly - in another PR

File: Gemfile.lock [hunk 2]
```diff
@@ -394,4 +394,4 @@ RUBY VERSION
    ruby 2.7.2p137
 
 BUNDLED WITH
-   2.2.6
+   2.2.6
```
Comment (guy-roberts) [inline Gemfile.lock:397]: Can you check if there is a trailing space here, and if there is, remove it and push the change to github ? It makes the PR simpler

**Review by guy-roberts — COMMENTED** [review]
Looks good to me.

You might want to see if the last line of Gemfile.lock has a trailing space, and if so, remove it, to simplify the diffs

File: app/webpacker/packs/application.js [hunk 3]
```diff
@@ -15,10 +15,20 @@ $(document).ready(function() {
...
     createPreview('#markdown-editor','#markdown-render');
   });
-
+  
   //clipboard
   $('#clipboard_copier').click(function() {
     copyToClipboard();
   });
+
+  document.querySelectorAll('.nhsuk-card--clickable').forEach((panel) => {
```
Comment (tonyto) [inline app/webpacker/packs/application.js:24]: Worth nothing that this bit of JS is written differently to the  functions above it.

- Use of ES6 arrow syntax vs ES5 function declaration.
- Use of `querySelector` which supports IE 9 +

It'll be nice to be consistent and figure out if support for versions of IE < 9

/cc @marcrice09 @guy-roberts

**Review by guy-roberts — APPROVED** [review]

**Pull request merged**